tiering/memtier.c

```diff
@@ -131,6 +163,12 @@ MEMTIER_EXPORT int posix_memalign(void **memptr, size_t alignment, size_t size)
 
 MEMTIER_EXPORT void free(void *ptr)
 {
+    // TODO!!!
```

Comment:
This function is not implemented at all... Would be best to add some log, TODOs might be overlooked